ee/cis/macos-13/test/scripts/CIS_2.6.7.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -eu
+
+sudo security authorizationdb read system.preferences > /tmp/system.preferences.plist
+defaults write /tmp/system.preferences.plist shared -bool false
+sudo security authorizationdb write system.preferences < /tmp/system.preferences.plist
```

Comment:
Does this work?
Here's a note from the PDF for this check:
```
Note: In previous OS versions of the macOS Benchmarks, this has been an automated recommendation. In the initial release of macOS 13.0 Ventura, this setting does not apply properly. Once the setting starts applying properly, then the recommendation will move back to automated.
```

Reply:
It seems to work for me since it changes the checkbox, but I'm having trouble telling the difference between having it on and off. I tested with firewall because I thought that was something locked, but it turns out that's always locked

Reply:
Okay, looks like I had to switch to an unpriveleged account. It does work for me.